nifi-stateless/nifi-stateless-api/src/main/java/org/apache/nifi/stateless/config/PropertiesFileEngineConfigurationParser.java

```diff
@@ -53,6 +54,9 @@ public class PropertiesFileEngineConfigurationParser {
     private static final String CONTENT_REPO_DIRECTORY = PREFIX + "content.repository.directory";
     private static final String STATUS_TASK_INTERVAL = PREFIX + "status.task.interval";
 
+    private static final String COMPONENT_ENABLE_TIMEOUT = PREFIX + "component.enableTimeout";
+    private static final String PROCESSOR_START_TIMEOUT = PREFIX + "processor.startTimeout";
```

Comment:
We should use dotted names instead of camel-case - `component.enable.timeout` and `processor.start.timeout` rather than `enableTimeout` and `startTimeout`

Reply:
Fair enough.